app/views/authority/show.html.haml

```diff
@@ -28,7 +28,7 @@
         ga('send', {
           hitType: 'event',
           eventCategory: 'Show by',
-          eventAction: 'Organisation - #{@collection.name.html_safe} - ' + event.target.innerText,
+          eventAction: 'Organisation - #{@collection.name.html_safe} - ' + event.target.innerText.replace(/ register/, ''),
```

Comment:
can we use end of string regex i.e. `^` to make sure this does not erroneously do the wrong replace?